docs/modules/automation-testing/pages/scripting/run-applitools-eyes-script.adoc

```diff
@@ -0,0 +1,151 @@
+
+== Add custom domain name for the Kobiton Portal
+
+[NOTE]
+This section is required for Standalone/On-prem customers only.
+
+Set up a custom domain for the Kobiton Portal (the Dell server) that ends with `kobiton.com` on the script runner machine. You can do this by either:
+
+* Adding an entry in the DNS server in your organization.
+
+* Create a new entry like the below in the local `/etc/hosts` (Linux, MacOS) or `c:\Windows\System32\Drivers\etc\hosts` (Windows) of the script runner machine.
+
++
+
+[source]
+<Dell server IP address> example.kobiton.com
+
+
+Example of qualified domain name: `example.kobiton.com`, `lab1.kobiton.com`.
+
+== Add server URL and capabilities to your script
+
+To run on Kobiton devices, add the server URL and capabilities to your Appium script. See xref:capabilities/auto-generate-capabilities.adoc[this guide] for more details.
+
+The following example is based on an Android test script written in Java from the link:https://applitools.com/tutorials/quickstart/native-mobile/appium/java[Applitools Eyes SDK tutorial page]. Alter the script from the tutorial in 3 places:
+
+*  Capabilities:
++
+[source,javascript]
+----
+// Set desired capabilities.
+DesiredCapabilities capabilities = new DesiredCapabilities();
+
+  capabilities.setCapability("platformName", "Android");
+  // Use either deviceName or udid to select the device
+  // capabilities.setCapability("deviceName", "Pixel 7");
+  capabilities.setCapability("udid", "f40d06b2-daa4-4b62-896e-c0e675135ecd ");
+  capabilities.setCapability("platformVersion", "*");
+  capabilities.setCapability("app", "https://applitoolsnmlresources.z19.web.core.windows.net/TestApps/app-debug.apk");
+  capabilities.setCapability("browserName", "");
+  capabilities.setCapability("automationName", "UiAutomator2");
+  // Use the kobiton:runtime capability below if you want to launch a Basic Appium 2 session
+  // capabilities.setCapability("kobiton:runtime", "appium");
+----
+
+* Kobiton server URL:
++
+[source,javascript]
+----
+// Open the app.
+WebDriver driver = new AndroidDriver(new URL("https://[username]:[api_key]@<portal_domain>/wd/hub"), capabilities);
+----
+** Replace `[portal_domain]` with either:
+
+*** `api.kobiton.com` for Cloud/Hybrid customers.
+
+*** `[custom_domain]:3000` for Standalone/Hybrid customer, where `[custom_domain]` is the domain specified in xref:#_add_custom_domain_name_for_the_kobiton_portal[this section]. __Example__: `example.kobiton.com:3000`.
+
+** Replace `[username]` and `[api_key]` with your Kobiton account details.
+
+* Additionally, replace `YOUR_API_KEY` below with the runner API key from Applitools.
++
+[source,javascript]
+----
+// Initialize the eyes SDK and set your private API key.
+Eyes eyes = new Eyes();
+eyes.setApiKey("YOUR_API_KEY");
+----
+
+At the time of writing, the final test script would look like the below:

```

Comment:
Remove "At the time of writing," and make the capital "The final..."
(The phrase is not needed and instills doubt. All tech docs are valid as of the time of writing.)